### PR TITLE
Add budget information at project level

### PR DESCRIPTION
**Pull request by ketlane**
### Ticket
Link to ticket: https://teamleader.atlassian.net/browse/PRO-1979

### What has been done
- Added budget information at project level
- Documented the changes in the changelog

### CODE REVIEW

**Review by joaoinacio — DISMISSED** [review]

**Review by ArnaudWeyts — DISMISSED** [review]

**Review by adognameddog — DISMISSED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -371,12 +371,19 @@ We list all backwards-compatible additions here. These are currently available i
 
 #### Latest
 
+- We added `budget` to `projects.list` and `projects.info`.
```
Comment (stivni) [inline apiary.apib]: You don't need to document it twice... adding it to the latest section is fine.
Every month we will group "latest" stuff to the specific month, so those entries stay stable and people don't have to reread them all the time..

**Review by stivni — DISMISSED** [review]
The only remark I have, is that the provided budget is in theory the only updateable field.
So if we want to add that to an API, we should think about how we do that, so that that is clear...
(Is this planned, for sake of completeness of the API?)

**Review by jenssegers — DISMISSED** [review]

**Review by joaoinacio — APPROVED** [review]

**Review by ArnaudWeyts — APPROVED** [review]

**Comment by ketlane** [comment]
> The only remark I have, is that the provided budget is in theory the only updateable field.
> So if we want to add that to an API, we should think about how we do that, so that that is clear...
> (Is this planned, for sake of completeness of the API?)

The old API allows to specify a key called `project_budget` when creating a project. The new API does not currently allow to specify the budget. I propose to use a key called `budget`. If we go with that, I think it makes sense that `projects.info` returns a key called `budget.provided` (as it does now).

However, we haven't planned to allow specifying a budget when adding a project in API v2. I guess we could include that in the near future (with a way to also update this value when updating the project), it's not much work and should close this loop.

**Review by jenssegers — APPROVED** [review]

**Review by adognameddog — APPROVED** [review]

**Comment by ketlane** [comment]
Closed in favour of https://github.com/teamleadercrm/api/pull/454

**Pull request closed without merging**